[2024-06-20T11:43:23.199+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-20T11:38:15.938449+00:00 [queued]>
[2024-06-20T11:43:23.203+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-20T11:38:15.938449+00:00 [queued]>
[2024-06-20T11:43:23.203+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2024-06-20T11:43:23.210+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2024-06-20 11:38:15.938449+00:00
[2024-06-20T11:43:23.213+0000] {standard_task_runner.py:57} INFO - Started process 34400 to run task
[2024-06-20T11:43:23.216+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'scheduled__2024-06-20T11:38:15.938449+00:00', '--job-id', '1948', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmpeiajxzur']
[2024-06-20T11:43:23.216+0000] {standard_task_runner.py:85} INFO - Job 1948: Subtask kafka_data_stream
[2024-06-20T11:43:23.231+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 1948 for task kafka_data_stream (DagRun for kafka_spark_dag with run_id or execution_date of 'scheduled__2024-06-20T11:38:15.938449+00:00' not found; 34400)
[2024-06-20T11:43:23.234+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-06-20T11:43:23.247+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
